stress/common/src/main/java/alluxio/stress/fuse/FuseIOOperation.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.fuse;
+
+/**
+ * The operations for the FUSE stress tests.
+ */
+public enum FuseIOOperation {
+  /** The write operation, for writing data for the read operations. */
```

Comment:
Write operation to test the write throughput or prepare data for reading

Reply:
Depending on how we are going to do with testing write throughput, we may be separate the data preparation part with this write, but this is for the next step